lib/matplotlib/ticker.py

```diff
@@ -499,13 +482,37 @@ class ScalarFormatter(Formatter):
     In addition to the parameters above, the formatting of scientific vs.
     floating point representation can be configured via `.set_scientific`
     and `.set_powerlimits`).
+
+    **Offset notation and scientific notation**
+
+    Offset notation and scientific notation look quite similar at first sight.
+    Both split some information from the formatted tick values and display it
+    at the end of the axis.
+
+    - The scientific notation splits up the order of magnitude, i.e. a
+      multiplicative scaling factor, e.g. ``1e6``.
+
+    - The offset notation separates an additive constant, e.g. ``+1e6``. The
+      offset notation label is always prefixed with a ``+`` or ``-`` sign
+      and is thus distinguishable from the order of magnitude label.
+
+    Example:
```

Comment:
Can be titled?
```suggestion
    Examples
    --------
```

Reply:
Technically, yes. But I rather see this as an Illustration of the offset and scientific notation „subsection“ of Notes. The Examples section would be for a more general example.

Reply:
Technically, yes. But I rather see this as an Illustration of the offset and scientific notation „subsection“ of Notes. The Examples section would be for a more general example.

Reply:
Wrote an explanatory paragraph instead of just stating "Example".